encoder/vaapiencoder_h264.cpp

```diff
@@ -1893,6 +1893,7 @@ bool VaapiEncoderH264::addSliceHeaders (const PicturePtr& picture) const
         sliceParam->idr_pic_id = m_idrNum;
         sliceParam->pic_order_cnt_lsb = picture->m_poc % m_maxPicOrderCnt;
 
+        sliceParam->direct_spatial_mv_pred_flag = 1;
```

Comment:
yes, At least I don't know why. But it can fix this issue.
According to the driver's comment, it can make better quality.